### PR TITLE
Feature/add stats to mobile remote video menu

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

**Comment by tmoldovan8x8** [comment]
Depends on the changes in lib-jitsi-meet: https://github.com/jitsi/lib-jitsi-meet/pull/1437

**Comment by saghul** [comment]
Jenkins please test this please.

File: react/features/remote-video-menu/components/native/ConnectionStatusComponent.js [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+import styles from './styles';
+
+/**
+ * Size of the rendered avatar in the menu.
+ */
+const AVATAR_SIZE = 25;
+
+const CONNECTION_QUALITY = [
+    'Low',
```
Comment (saghul) [inline react/features/remote-video-menu/components/native/ConnectionStatusComponent.js:25]: Let's reuse the `connectionindicator.quality.XXX` translations instead,

File: react/features/remote-video-menu/components/native/ConnectionStatusComponent.js [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._onStatsUpdated = this._onStatsUpdated.bind(this);
+        this._onCancel = this._onCancel.bind(this);
+        this._renderMenuHeader = this._renderMenuHeader.bind(this);
+
+        this.state = {
+            resolutionString: 'N/A',
```
Comment (saghul) [inline react/features/remote-video-menu/components/native/ConnectionStatusComponent.js:101]: please sort these alphabetically

**Review by saghul — APPROVED** [review]

**Pull request merged**